src/corg/coding-and-design-guidelines/code-review-guidelines.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Code Review Guidelines
+review:
+    comment: ''
+    date: ''
+    status: ok
+tree_item_index: 300
+history:
+    -
+        author: Florent Guillaume
+        date: '2018-05-11 18:03'
+        message: ''
+        version: '1'
+---
+Code reviews are a collaborative process between coders and reviewers — this is not a battle. The goal of the reviews is to improve the code quality by having several pairs of eyes on the code, for the benefit of all.
+
+## When creating a Pull Request
+
+### Follow the coding standards
+
+Important coding standards are described in the [Java Code Style]({{page page='java-code-style'}}) and [JavaScript Code Style]({{page page='javascript-code-style'}}) pages.
+
+### Design for easy review
+
+The code must be designed for easy review and for future readers of the commits:
+- Don't mix cleanups/reformatting with actual code.
+- Use several commits for unrelated work.
+- Use only one commit for related thing. For example don't have a separate "fix typo" commit if the typo fixed is in another commit of the Pull Request.
+- Diffs must be minimal.
```

Comment:
add a sample

optionally also link to related Git usages such as https://gist.github.com/jcarsique/359400aab7403428e2a6703276f8e544

Reply:
Not sure what sample you would like to see, please provide an example.